package.json

```diff
@@ -30,6 +30,7 @@
     "@types/chance": "^1.1.3",
     "aws-sdk": "^2.711.0",
     "import-all.macro": "^3.1.0",
+    "why": "^0.1.12",
```

Comment:
why?

Reply:
Just used this to investigate a vulnerable dependency issue - will remove it now.